libs/sdk-ui-kit/src/Messages/useToastMessage.ts

```diff
@@ -0,0 +1,45 @@
+// (C) 2021 GoodData Corporation
+
+import { useContext } from "react";
+import { MessageDescriptor, useIntl } from "react-intl";
+import { ToastMessageContext } from "./ToastMessageContext";
+import { MessageType } from "./typings";
+
+/**
+ * @public
+ */
+export type AddMessageType = (message: MessageDescriptor) => void;
+
+/**
+ * @public
+ */
+export interface UseToastMessageType {
+    addSuccess: AddMessageType;
+    addProgress: AddMessageType;
+    addWarning: AddMessageType;
+    addError: AddMessageType;
+}
+
+/**
+ * @public
```

Comment:
Please change this and all others in kit to internal, everything in kit should be internal